Add ESP32 support (#28)
cloud4rpi/docs: docs/faq.md

> Frequently asked questions and possible issues you may face when working with [Cloud4RPi](https://cloud4rpi.io):

Reviewer: ```suggestion
Frequently asked questions and possible issues you can encounter with [Cloud4RPi](https://cloud4rpi.io):
```

    ![](/res/ds18b20.png)

## How to avoid the "Insecure Platform" warning?

> The `InsecurePlatformWarning: A true SSLContext object is not available...` notification appears when you run Cloud4RPi on a Python v2.7.9 and earlier. Use the following command to check the Python version:

Reviewer: ```suggestion
The `InsecurePlatformWarning: A true SSLContext object is not available...` notification appears when you run Cloud4RPi on Python v2.7.9 and earlier. Use the following command to check the Python version:
```

> You can use [one of the following approaches](https://docs.python.org/2/using/index.html) to install the latest version.

Reviewer: ```suggestion
You can use [one of the following approaches](https://docs.python.org/2/using/index.html) to install the latest version:
```

## Where are the log files?

> Use one of the following commands to read the logs containing the service's output and errors:

Reviewer: ```suggestion
Use one of the following commands to read the logs that contain the service's output and errors:
```

!!! Note
    The variable update interval does not affect commands, and variable changes made through the Web UI (**Switch** or **Slider** widget) are applied instantly.

> ## How to avoid WiFi library collision on Arduino?

Reviewer 2: It's mean "Arduino IDE" not framework. May be better to use non-official name?

> The `Multiple libraries were found for "WiFi.h"` message may appear in Arduino IDE, which means the IDE has two libraries with the same name, and it can not decide which of them to use.

Reviewer: ```suggestion
The `Multiple libraries were found for "WiFi.h"` message can appear in Arduino IDE. This indicates the IDE has two libraries with the same name, and it does not know which one to use.
```

> You can avoid this error by specifying the absolute path to the header file in the `#include` directive, for example, like this:

Reviewer: ```suggestion
To avoid this error, specify the absolute path to the header file in the `#include` directive, for example:
```

```c
#include <C:\Users\user\Documents\ArduinoData\packages\esp32\hardware\esp32\1.0.2\libraries\WiFi\src\WiFi.h>
```

> The valid path to both libraries appear in the error message, you can copy-paste it. Remember to add the `src\` subfolder between the library path and header file name.

Reviewer: ```suggestion
You can copy-paste the valid path to both libraries that appear in the error message. Add the `src\` subfolder between the library path and header file name.
```
